app/src/main/java/com/example/househomey/ItemAdapter.java

```diff
@@ -44,6 +54,29 @@ public View getView(int position, @Nullable View convertView, @NonNull ViewGroup
         String dateCost = new SimpleDateFormat("yyyy-MM-dd").format(item.getAcquisitionDate()) + " | $" + item.getCost();
         ((TextView) view.findViewById(R.id.item_text)).setText(dateCost);
 
+        Button viewItemButton = view.findViewById(R.id.action_view);
+        FragmentManager fragmentManager = ((AppCompatActivity) context).getSupportFragmentManager();
+        viewItemButton.setOnClickListener(v -> {
+
+            ViewItemFragment viewItemFragment = new ViewItemFragment();
+            Bundle details = new Bundle();
+            details.putString("make", item.getMake());
+            details.putString("model", item.getModel());
+            details.putString("serialNumber", item.getSerialNumber());
+            details.putString("cost", item.getCost().toString());
+            details.putString("comment", item.getComment());
+            viewItemFragment.setArguments(details);
+
+            // Begin a fragment transaction
+            FragmentTransaction transaction = fragmentManager.beginTransaction();
```

Comment:
@owencooke Not sure if ```navigateToFragment()``` from main activity could be made static and used here and consolidated with the ```returnToHomeScreen()``` from AddItemFragment. Might be worth looking into

Reply:
Yep good idea. I can look into adding that in my next PR